Documentation/freebsd-instructions.md

```diff
@@ -42,9 +64,7 @@ Build the Runtime
 
 To build the runtime on FreeBSD, run build.sh from the root of the coreclr repository:
 
-```
-janhenke@freebsd-frankfurt:~/git/coreclr % ./build.sh
-```
+```janhenke@freebsd-frankfurt:~/git/coreclr % ./build.sh```
```

Comment:
This line *was* a one-liner but was nit-picked to use the same formatting as the rest of the document.